emails/students/7-endeval.txt

```diff
@@ -0,0 +1,16 @@
+Subject: Endterm Evaluations are due January 7th
+Body:
+
+Dear Student,
+
+KWoC 2017 is almost over!
+
+At this point, you should be done with all your contributions to the projects. Now is a good time to clean up your code, update documentation and follow your mentor’s instructions about getting your code ready for final review.
+
+Be sure to complete your Final Evaluation before Sunday, January 1st at 23:59 IST. You can access it from this link :
```

Comment:
@OrkoHunter Shouldn't this be January 7th?

Reply:
Yes! Thank you :D , I'll push the change.

Reply:
I think the data hasn't been changed. Thankfully, I noticed this comment before I sent out the mails.

Reply:
Since this PR has been merged, I guess edits won't show up here, even if a commit in the PR is amended. I think @OrkoHunter fixed it by pushing to master.

Reply:
I fetched the master branch. This line wasn't changed. The subject was though.

Reply:
Excuse me guys. I committed the change to push it directly for the master, but then a cute baby knocked my door and I was playing with her for half an hour. I thought I pushed but I didn't. :D